docs/pxe/maas.md

```diff
@@ -0,0 +1,172 @@
+1. Look for the fabric whose subnet matches your internal network (`192.168.1.0/24` in our two-VM example). Then, in this row, click the "untagged" link in the "VLAN" column.
+1. This should take you to the page for the Default VLAN on your internal network. The second panel in this page will be for DCHP, and there should be a button all the way to the right labeled "Enable DHCP".
+    Click it.
+1. On the DHCP configuration page, you will be asked for the configuration of the "Reserved dynamic range". On a real network, you should ensure that the gateway IP is correct, and that the IPs in this range don't overlap with any IPs your site is using. For our example, we will accept the defaults and click "Configure DHCP".
+
+
+## Booting the test VM from MAAS
+
+### Adding the test VM to the MAAS database
+
+Once MAAS is set up to run DHCP on the internal network, any other machine which is configured to PXE boot should automatically boot from MAAS.
+If it isn't a machine which MAAS already knows about, it will boot into the [MAAS enlistment image](https://maas.io/docs/add-machines#heading--enlistment),
+which will detect some basic information about the machine and load it into the MAAS database.
+
+To demonstrate this, ensure the second server (`test-vm`) is configured to PXE boot, then boot it up and take a look at its console.
+You should see the server PXE boot, and then the console will start scrolling with messages as it loads the MAAS enlistment image.
+After a little while, it will show a login prompt that shows its name as `maas-enlisting-node`, and then continue scrolling through several other messages.
+After another few minutes, the node will switch off!
+
+### Configuring the test VM for power control (or manual)
+
+In the MAAS web interface, click the "Machines" link in the navigation bar.
+The list of machines should now include one entry.
+The name of the new machine will be auto-generated from an adjective and an animal, for example "alert-pigeon".
+
+Click the name of the new machine, and this will navigate to a page for this machine.
+It should show several auto-detected details about the machine, such as the nymber of CPUs, the memory, and the storage available on the machine.
+If you want to, you can also rename the machine by clicking the name at the top of the page.
+
+Before we can deploy this machine, we have to configure it so that MAAS can turn it on or off (or tell MAAS that we will do this manually).
+
+1. On the page for the machine, look for the navigation bar *under* the machine's name.
+1. Click Configuration.
+1. Scroll down until you see the Power configuration setting.
+    If this has already been selected to a pre-configured value, you're good to go, because MAAS has autodetected how to control the power on this node!
+1. If it's not yet selected, click the drop-down menu and it will show several different types of power control available.
+    For a physical machine, the most common choice will be "IPMI". 
```

Comment:
Might be useful to add that if the configuration is "IPMI" (ex: for DGX) then the user should also enter the BMC login creds.